lib/src/http.rs

```diff
@@ -1641,132 +1728,63 @@ impl Listener {
-                debug!("{} remove cluster {:?}", message.id, cluster_id);
-                self.remove_cluster(&cluster_id);
-                ProxyResponse::ok(message.id)
+                debug!("{} remove cluster {:?}", request_id, cluster_id);
+                self.remove_cluster(&cluster_id)
             }
             ProxyRequestOrder::AddHttpFrontend(front) => {
-                debug!("{} add front {:?}", message.id, front);
-                if let Some(listener) = self
-                    .listeners
-                    .values()
-                    .find(|l| l.borrow().address == front.address)
-                {
-                    let mut owned = listener.borrow_mut();
-
-                    let hostname = front.hostname.to_owned();
-                    let tags = front.tags.to_owned();
-
-                    match owned.add_http_front(front) {
-                        Ok(_) => {
-                            owned.set_tags(hostname, tags);
-
-                            ProxyResponse::ok(message.id)
-                        }
-                        Err(err) => ProxyResponse::error(message.id, err),
-                    }
-                } else {
-                    ProxyResponse::error(
-                        message.id,
-                        format!("no HTTP listener found for front: {:?}", front),
-                    )
-
-                    // let (listener, tokens) = Listener::new(HttpListener::default(), event_loop,
-                    //  self.pool.clone(), None, token: Token) -> (Listener,HashSet<Token>
-                }
+                debug!("{} add front {:?}", request_id, front);
+                self.add_http_frontend(front)
             }
             ProxyRequestOrder::RemoveHttpFrontend(front) => {
-                debug!("{} front {:?}", message.id, front);
-                if let Some(listener) = self
-                    .listeners
-                    .values()
-                    .find(|l| l.borrow().address == front.address)
-                {
-                    let mut owned = listener.borrow_mut();
-
-                    let hostname = front.hostname.to_owned();
-
-                    match owned.remove_http_front(front) {
-                        Ok(_) => {
-                            owned.set_tags(hostname, None);
-                            ProxyResponse::ok(message.id)
-                        }
-                        Err(err) => ProxyResponse::error(message.id, err),
-                    }
-                } else {
-                    ProxyResponse::error(
-                        message.id,
-                        "trying to remove front from non existing listener",
-                    )
-                }
+                debug!("{} remove front {:?}", request_id, front);
+                self.remove_http_frontend(front)
             }
             ProxyRequestOrder::RemoveListener(remove) => {
                 debug!("removing HTTP listener at address {:?}", remove.address);
```

Comment:
To discuss, but to me this kind of log could be an info one. To integrate into another pull request

Reply:
why not set everything to `info!` right away?

Reply:
It did change all these `debug!` statement to `info!` ones.

Reply:
Greate!